.gitignore

```diff
@@ -1,2 +1,3 @@
 /.idea/
 /.gradle/
+/build/
```

Comment:
Czy **.gitignore** zwykle zawiera tylko te trzy foldery, czy być to zależy od projektu? Tutaj zakładam, że ignorujemy te foldery, żeby nie zaśmiecać repozytorium i wrzucać wyłącznie istotny kod, który będzie podlegał "code review".

Reply:
To zupełnie zależy od projektu - te .gitignore potrafią być bardzo bardzo rozbudowane (tu np jest całe repozytorium z przydatnymi plikami per język programowania: https://github.com/github/gitignore). Sam IntelliJ też trochę pomaga (można kliknąć prawym na folderze -> git -> add to .gitignore). Ale też może być projekt gdzie (niektóre) pliki np z /.idea/ nie będą ignorowane bo zespół będzie chciał mieć jakieś propertiesy wspólne u każdego w IDE :) 

Co do tego czemu ignorujemy - dokładnie, zmiany w tych plikach nie mają żadnego wpływu na zadania, a IntelliJ automatycznie tam sobie jakieś zmiany generuje za każdym razem